components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts

```diff
@@ -106,6 +93,26 @@ export class CreateUserModalComponent implements OnInit, OnDestroy, OnChanges {
       filter(state => state === EntityStatus.loadingSuccess),
       takeUntil(this.isDestroyed))
       .subscribe(() => this.closeCreateModal());
+
+    combineLatest([
```

Comment:
hmm so this now subscribes directly to the store instead of getting it passes from the parent component (which was user-management). I like eliminating the middle man.